.github/workflows/stale.yml

```diff
@@ -24,7 +24,6 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'A friendly reminder that this issue had no activity for 30 days.'
         stale-pr-message: 'A friendly reminder that this PR had no activity for 30 days.'
-        stale-issue-label: 'stale-issue'
```

Comment:
according to the upstream doc[1] this defaults to `Stale` so I guess it will still add a label

[1] https://github.com/actions/stale#stale-issue-label

Reply:
Nice catch. I don't have another idea at the moment, so I am going to close.